cdap-app-templates/cdap-etl/cdap-data-streams-base/src/main/java/io/cdap/cdap/datastreams/DataStreamsPipelineSpecGenerator.java

```diff
@@ -118,6 +140,7 @@ void configureAtleastOnceMode(DataStreamsConfig config, DataStreamsPipelineSpec.
       DataStreamsStateSpec stateSpec = DataStreamsStateSpec.getBuilder(DataStreamsStateSpec.Mode.STATE_STORE)
         .build();
       specBuilder.setStateSpec(stateSpec);
+      emitMetrics(Constants.Metrics.AtleastOnceProcessing.ATLEAST_ONCE_PROCESSING_ON_STORE_COUNT);
```

Comment:
These metrics will be incremented for preview, deploy and pipeline runs. I think we will want these only for pipeline runs. Can we do this in the driver ?